spdmlib/src/message/capability.rs

```diff
@@ -54,7 +54,19 @@ impl SpdmCodec for SpdmGetCapabilitiesRequestPayload {
         context: &mut common::SpdmContext,
         r: &mut Reader,
     ) -> Option<SpdmGetCapabilitiesRequestPayload> {
-        u8::read(r)?; // param1
+        if context.negotiate_info.spdm_version_sel >= SpdmVersion::SpdmVersion13 {
```

Comment:
If the GET_CAPABILITIES request sets Bit 0 of Param1 to a value of 1 and the Responder
does not support the Large SPDM message transfer mechanism ( CHUNK_CAP=0 ), the Responder shall send an
ERROR message of ErrorCode=InvalidRequest .